NSS-Day-Cohort-42/rare-media-moguls
# Tag Provider functional.

#### Changes Made
1. Added file `TagProvider.js` to `tags` directory.
1. Modified `ApplicationViews.js` to import `TagProvider`

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-tag-provider
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> No changes expected
6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
